FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+std_msgs::Int8 percent_full;
+ros::Publisher pub_percent_full("percent_full", &percent_full);
+ros::NodeHandle nh;
+
+void setup()
+{
+  nh.initNode();
+  Serial.begin(9600);
+  nh.getHardware()->setBaud(9600);
+  nh.advertise(pub_percent_full);
+}
+
+long publisher_timer;
+const int READ_PIN = A7;
+
+void loop() {
+  if (millis() > publisher_timer) {
```

Comment:
Look at this file lines 295 to 305. 
https://github.com/Greenzie/interface_board/blob/main/intrepid/p5/src/main.ino

You have  a clever way of incrementing the timing of this publisher but it isn't typically how we do things. 

Can you try and implement somehting like we have in the above link

last_publish_time
variable to define  publish loop time

then compare to millis()

Reply:
Got it :+1:

Reply:
Okay this should be fixed. Thanks